themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -58,27 +58,17 @@ VuFind.register('itemStatuses', function ItemStatuses() {
       el.querySelectorAll('.callnumber,.hideIfDetailed,.location').forEach((e) => e.classList.add('hidden'));
       var locationListHTML = "";
       for (var x = 0; x < result.locationList.length; x++) {
+        var status = result.locationList[x].status;
+        var locationClass =
+          availabilityClasses[status] === 'true' ? 'available'
```

Comment:
The `availabilityClasses` keys are the "availabilityAsString" values. So `availabilityClasses[availability]` should be sufficient. For the status `availability === 'true'` and `availability === 'false'` need to be checked.